spec/unit/outputs/elasticsearch/data_stream_support_spec.rb

```diff
@@ -152,6 +152,42 @@
 
   end
 
+  context 'ds value-dependent configuration' do
+    # Valid settings values
+    let(:options) { super().merge(
+      'action' => 'create',
+      'routing' => 'any',
+      'pipeline' => 'any',
```

Comment:
Are `routing` and `pipeline` required for the the test?

Reply:
I'd say yes, this test is checking the `invalid_data_stream_params` method conditions (which is currently uncovered),
Having `routing` and `pipeline` here will ensure we do not disable the data stream if any of them is set on the plugin configuration.

Reply:
oks